metaapp/src/main/java/com/sourcepointmeta/app/ui/ConsentViewActivity.java

```diff
@@ -93,7 +93,7 @@ private void removeWebView(View view) {
     private GDPRConsentLib buildConsentLib(Property property, Activity activity) {
         ConsentLibBuilder consentLibBuilder = GDPRConsentLib.newBuilder(property.getAccountID(), property.getProperty(), property.getPropertyID(), property.getPmID(), activity)
                 .setStagingCampaign(property.isStaging())
-                //.setMessageTimeOut(15000)
+                .setMessageTimeOut(30000)
```

Comment:
@Suryakantj I think it would be better to have a final static field for this default value. What do you think?

Reply:
Good suggestion.